### PR TITLE
redeployed upgraded contracts 0.8.0

### PR DESCRIPTION
**Pull request by adi44**

### CODE REVIEW

File: .env.tpl [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Seed phrase from which accounts are derived
-MNEMONIC=
+MNEMONIC="winter sunset nice emerge melt punch antique strike output depend crumble grow"
```
Comment (SkandaBhat) [inline .env.tpl:2]: Mnemonic and other details should not be here

File: hardhat.config.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ module.exports = {
       accounts: { mnemonic: MNEMONIC },
       chainId: ENV_CHAIN_IDS[NETWORK],
       gas: 7700000,
-      gasPrice: 1 * GWEI,
+      gasPrice: 10 * GWEI,
```
Comment (SkandaBhat) [inline hardhat.config.js:63]: This change should be on a different PR since it is doing something other than the PR title.

File: migrations/src/postDeploymentSetup.js [hunk 3]
```diff
@@ -54,7 +54,7 @@ module.exports = async () => {
 
   console.log("Waiting for post-deployment setup transactions to get confirmed");
   for (let i = 0; i < pendingTransactions.length; i++) {
-    pendingTransactions[i].wait();
+    await pendingTransactions[i].wait();
```
Comment (SkandaBhat) [inline migrations/src/postDeploymentSetup.js:57]: This change should be in another PR.

**Comment by adi44** [comment]
Making different PRs for different changes

File: hardhat.config.js (hunk 2 above)
Comment (dev1644) [reply hardhat.config.js:63]: Yeah, this isn't even needed, `1 GWEI` is fine on testnet, even if you want to experiment with high gas prices, you can just modify it. Please revert it. @adi44

Comment (adi44) [reply hardhat.config.js:63]: https://github.com/razor-network/contracts/pull/96 @dev1644 it is the latest PR
